cli/src/action/registry/mod.rs

```diff
@@ -16,7 +16,7 @@
 mod api;
 
 use clap::ArgMatches;
-use splinter::registry::Node;
+use splinter::registry::{Node, YamlNode};
```

Comment:
Typos in commit message: s/should be/shouldn't/ and s/handler/handle/

Reply:
Fixed